alamesa/src/components/Header.js

```diff
@@ -76,13 +79,13 @@ function Header() {
       <Navigation className="navigation">
         <NavigationMenu>
           <NavigationMenuItem className="navigation__menu-item restaurant">
-            <Anchor href="./">Restaurantes</Anchor>
+            <Anchor to="./">Restaurantes</Anchor>
```

Comment:
Acá no es necesario tener rutas relativas. Puedes dejar simplemente `to="/"`. Este comentario es valido para los siguientes Anchor también.

Otra cosa, en este caso estas definiendo los valores de los props con comillas dobles, pero en App los defines usando comillas simples. Es mejor usar un solo tipo (normalmente se usan comillas dobles en jsx)

P.s. las comillas entre el código `js` y `jsx` pueden ser diferentes. Ej: yo en js uso comillas simples y en jsx uso siempre comillas dobles